packages/database/lib/schema.ts

```diff
@@ -43,20 +56,29 @@ export const daos = pgTable('daos', {
   daoAcct: pubkey('dao_acct').primaryKey(),
   name: varchar('name').notNull(),
   url: varchar('url').notNull(),
+  xAccount: varchar('x_account'),
+  gitHub: varchar('github'),
+  description: text('description'),
   // In FaaS, each DAO is tied to its own token which futarchic markets will aim to pomp to the moon
-  mintAcct: pubkey('mint_acct').references(() => tokens.mintAcct).notNull()
+  mintAcct: pubkey('mint_acct').references(() => tokens.mintAcct).notNull(),
+  createdAt: timestamp('created_at').notNull().default(sql`now()`),
+  updatedAt: timestamp('updated_at').notNull().default(sql`now()`),
 });
 
 export const proposals = pgTable('proposals', {
   proposalAcct: pubkey('proposal_acct').primaryKey(),
   daoAcct: pubkey('dao_acct').references(() => daos.daoAcct).notNull(),
   proposalNum: bigint('proposal_num', {mode: 'bigint'}).notNull(),
+  title: varchar('title'),
```

Comment:
We'll go in and add titles for the historical proposals, and in theory when they're created from here on out it SHOULD be through the UI, but isn't strictly enforced, so I kept them optional...